yt_dlp/extractor/ilpost.py

```diff
@@ -0,0 +1,61 @@
+import functools
+
+from .common import InfoExtractor
+from ..utils import ExtractorError, float_or_none, int_or_none, url_or_none, urlencode_postdata
+from ..utils.traversal import traverse_obj
+
+
+class IlPostExtractorIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?ilpost\.it/episodes/(?P<id>[^/?]+).*'
+    _TESTS = [{
+        'url': 'https://www.ilpost.it/episodes/1-avis-akvasas-ka/',
+        'md5': '43649f002d85e1c2f319bb478d479c40',
+        'info_dict': {
+            'id': '2972047',
+            'ext': 'mp3',
+            'display_id': '1-avis-akvasas-ka',
+            'title': '1. Avis akvasas ka',
+            'url': 'https://www.ilpost.it/wp-content/uploads/2023/12/28/1703781217-l-invasione-pt1-v6.mp3',
+            'timestamp': 1703835014,
+            'upload_date': '20231229',
+            'duration': 2495.0,
+            'availability': 'public',
+            'series_id': '235598',
+            'description': '',
+        }
+    }]
+
+    def _real_extract(self, url):
+        podcast_display_id = self._match_id(url)
```

Comment:
`display_id` is the convention, shorter, and more distinct from `podcast_id`
```suggestion
        display_id = self._match_id(url)
```